images/applications/ts-ps-corrs.bash

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# CPU usage estimate (ps aux): 42.3%
+
+# Correctors
+/usr/local/bin/sirius-ioc-as-ps.py LA-RaCtrl:CO-PSCtrl-TS &> /ioc-logs/sirius-ioc-ts-ps-corrs & # 40.2%
+
+# Diagnostics
+/usr/local/bin/sirius-ioc-as-ps-diag.py TS ".*" ".*" &> /ioc-logs/sirius-ioc-ts-ps-diag-corrs & # 2.1%
```

Comment:
the correct filter here is `TS ".*" "C.*"`

Reply:
fixed.